core/new-gui/package.json

```diff
@@ -126,6 +126,7 @@
     "@types/uuid": "~8.3.4",
     "@typescript-eslint/eslint-plugin": "~5.14.0",
     "@typescript-eslint/parser": "~5.14.0",
+    "cat": "^0.2.0",
```

Comment:
it's better to use a specific version `0.2.0` rather than prefixing `~` or `^`. We will change other libraries to use specific version as well later.

Reply:
Changed. I found `yarn add cat` will automatically add the prefix `^` to version `0.2.0.` I manually edited `package.json` and `yarn.lock` to use `0.2.0`